packages/ui/package.json

```diff
@@ -20,7 +20,10 @@
     "@chakra-ui/theme-tools": "1.1.8",
     "@emotion/react": "11.4.0",
     "@emotion/styled": "11.3.0",
-    "framer-motion": "4.1.17"
+    "@types/react-router-dom": "5.1.8",
+    "framer-motion": "4.1.17",
+    "moment": "2.29.1",
```

Comment:
🤔 I'm not sure if we should add another project dependency just to display a date in a particular format. Even in case, we would need some library in the future it shouldn't be moment.js. You can read more about it [on their official website](https://momentjs.com/docs/#/-project-status/).